batch.go

```diff
@@ -77,7 +77,7 @@ func (c *Client) CreateAndBuyBatchWithContext(ctx context.Context, in ...*Shipme
 
 // ListBatchesResult holds the results from the list insurances API.
 type ListBatchesResult struct {
-	Insurances []*Insurance `json:"batches,omitempty"`
+	Batches []*Batch `json:"batches,omitempty"`
```

Comment:
Please break the batch changes into a separate PR. PRs should be as small as possible and only contain changes related to one thing. This PR is for the `refund` object and should only have changes related to that.